src/utils/buildExtractStylesLoader.js

```diff
@@ -35,5 +35,5 @@ Error: ${error}
       .map(loader => `${loader}!`)
       .join('')
   );
-  return ExtractTextPlugin.extract({ fallbackLoader, loader: restLoaders });
+  return ExtractTextPlugin.extract({ fallback: fallbackLoader, use: restLoaders });
```

Comment:
This PR currently doesn't solve that problem. A string is required by our own code. I just created an amendment that will solve this problem and be completely backwards compatible.